addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -4565,7 +4568,7 @@ public OnClientDisconnect(client)
 	FF2flags[client]=0;
 	Damage[client]=0;
 	uberTarget[client]=-1;
-	playBGM[client]=true;
+	playBGM[client]=false; // Better set it false
```

Comment:
See OnClientPostAdminCheck(client) and playBGM[0] and wen playBGM[client] become true